src/js/view/utils/submenu-factory.js

```diff
@@ -83,7 +91,7 @@ export function addQualitiesSubmenu(settingsMenu, qualitiesList, action, initial
         });
     });
 
-    const qualitiesSubmenu = makeSubmenu(settingsMenu, QUALITIES_SUBMENU, qualitiesItems, cloneIcon('quality-100'), tooltipText);
+    const qualitiesSubmenu = makeSubmenu(settingsMenu, QUALITIES_SUBMENU, qualitiesItems, cloneIcon('quality-100'), tooltipText, qualityText);
```

Comment:
localization.hd is being passed into `tooltipText` and localization.quality is being passed to `qualityText` yet they use the same string 'quality'. Is one deprecated ? @aliciahurst

Reply:
Wasn't aware that .quality existed. Only .hd is documented. It's intended that they're mapped to the same key, so yes, it seems we don't need .quality.